DevelopGitHub/GitHubExtension/GitHubExtension.Activity.External.Tests/Extensions/IdentityExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Security.Claims;
+using System.Security.Principal;
+using System.Web.Http;
+
+namespace GitHubExtension.Activity.External.Tests.Extensions
+{
+    public static class IdentityExtensions
```

Comment:
It is an extension for mocking identity in other our test projects it is called this way